LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/ModuleConnection/ModuleConnectionHost.cs

```diff
@@ -204,6 +207,22 @@ private bool TryUpdateConfigurationWithDesiredProperties(TwinCollection desiredP
             return false;
         }
 
+        private void UpdateProcessingDelayWithDesiredProperties(TwinCollection desiredProperties)
+        {
+            if (desiredProperties is null)
+            {
+                return;
+            }
+
+            var reader = new TwinCollectionReader(desiredProperties, this.logger);
+            if (reader.TryRead<int>(Constants.ProcessingDelayKey, out var processingDelay))
+            {
+                this.logger.LogDebug($"Updating processing delay for LNS {this.networkServerConfiguration.GatewayID} " +
```

Comment:
no need to add the GatewayID, the gatewayId is included in all entries, it's the host name.

Reply:
can we use structured logging for the delay. That allows to query for the entry when it is for example > 400.